packages/schema/src/config/common.ts

```diff
@@ -154,6 +154,13 @@ export default defineUntypedSchema({
     $resolve: async (val: string | undefined, get): Promise<string> => resolve(await get('rootDir') as string, val || '.nuxt'),
   },
 
+  /**
+   * For multi-app projects, the unique name of the Nuxt application.
+   */
+  appId: {
+    $resolve: (val: string) => val ?? 'nuxt-app',
```

Comment:
Don't we have a risk of name collision in case `appId` is not filled in in user land and default value `nuxt-app` is used multiple times?

Reply:
well, I would add Symbol('nuxt-app') so that the name does not conflict